core/trino-parser/src/main/java/io/trino/sql/SqlFormatter.java

```diff
@@ -289,29 +287,24 @@ protected Void visitQuery(Query node, Integer indent)
                 while (queries.hasNext()) {
                     WithQuery query = queries.next();
                     append(indent, formatExpression(query.getName()));
-                    query.getColumnNames().ifPresent(columnNames -> appendAliasColumns(builder, columnNames));
+                    query.getColumnNames().ifPresent(columnNames ->
+                            appendAliasColumns(builder, columnNames));
                     builder.append(" AS ");
                     process(new TableSubquery(query.getQuery()), indent);
                     builder.append('\n');
                     if (queries.hasNext()) {
                         builder.append(", ");
                     }
                 }
-            }
+            });
 
             processRelation(node.getQueryBody(), indent);
-
-            if (node.getOrderBy().isPresent()) {
-                process(node.getOrderBy().get(), indent);
-            }
-
-            if (node.getOffset().isPresent()) {
-                process(node.getOffset().get(), indent);
-            }
-
-            if (node.getLimit().isPresent()) {
-                process(node.getLimit().get(), indent);
-            }
+            node.getOrderBy().ifPresent(orderBy ->
```

Comment:
Same, these are easier to read as
```java
node.getOrderBy().ifPresent(orderBy -> process(orderBy, indent));
node.getOffset().ifPresent(offset -> process(offset, indent));
node.getLimit().ifPresent(limit -> process(limit, indent));
```
No need to wrap very short lines

Reply:
Sure, I'll fix that :) Thanks!